src/sentry/static/sentry/app/components/organizations/homeSidebar.jsx

```diff
@@ -33,15 +33,15 @@ const OrgSettingsMenu = ({access, org, features}) => {
       <h6 className="nav-header with-divider">{t('Manage')}</h6>
       <ul className="nav nav-stacked">
         {access.has('org:read') && (
-          <RouterOrBrowserLink isRouter={hasNewSettings} path={`${pathPrefix}/members/`}>
+          <ListLink to={`${pathPrefix}/members/`}>
```

Comment:
So IIUC, this will send them to the old path, which will return a `react_page_view`. But how will react know it has to render the new component on the old path?

Reply:
react knows because of react's router. 

So on the initial page load it goes django router -> `react_page_view` -> `app/main.jsx`/react router -> members.

If you're asking because it's not in the diff, that's because the route already exists, the navigation to it was just never exposed. i.e. all links to the Members view was using a browser link instead of a react router link.

Reply:
Oh I see. So any non-`react.html` django templates don't load *any* react components at all, including <Route> configuration, correct?